cms/templates/settings.html

```diff
@@ -380,6 +380,7 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                   % endif
                 % endif
 
+                % if about_page_editable:
```

Comment:
Might want to consider the `Introducing Your Course` header too - if we just hide the Card Image section and it has no other sections, that header alone is going to look weird.

Reply:
(See https://studio.edx.org/settings/details/course-v1:UBCx+IndEdu200x+1T2019 for how that section looks on stage right now - image is only thing in it.)